spec/views/spree/orders/edit.html.haml_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "spree/orders/edit.html.haml" do
+  let(:order) { create(:completed_order_with_fees) }
+  before do
+    assign(:order, order)
+    assign(:insufficient_stock_lines, [])
+    allow(view).to receive_messages(
+      order: order,
+      current_order: order,
+      pickup_time: 'time',
+      spree_current_user: create(:user),
+    )
+  end
+
+  describe "unit prices" do
+    it "displays unit prices informations if feature toggle is activated" do
+      allow(OpenFoodNetwork::FeatureToggle)
+      .to receive(:enabled?).with(:unit_price, anything) { true }
```

Comment:
multiline statements should be nested one level so readers notice that fact.

Reply:
You mean:
```ruby
allow(OpenFoodNetwork::FeatureToggle).to receive(:enabled?).with(:unit_price, anything) { true }
```
or
```ruby
    allow(OpenFoodNetwork::FeatureToggle)
      .to receive(:enabled?)
      .with(:unit_price, anything) { true }
```